packages/cashier/src/stores/withdraw-store.ts

```diff
@@ -281,7 +281,9 @@ export default class WithdrawStore {
     async check10kLimit() {
         const { client } = this.root_store;
 
-        const remainder = (await client.getLimits())?.get_limits?.remainder;
+        await this.WS.wait('get_limits');
```

Comment:
@wojciech-deriv We need data from `get_limits` endpoint. If you never call it, then you'll never get correct data... I've added this check here, because now this function called earlier than the same function in `client` store, and in result we made one more request in the `client` store in `init` method.

About your last question, suppose yes, we will wait for the response from needed endpoint. You can check this code here: https://github.com/deriv-com/deriv-api/blob/master/src/deriv_api/DerivAPIBasic.js. Our wait method is a wrapper for `expectResponse` method

Reply:
@wojciech-deriv Yes, you are right